.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  klever-publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: docker-build
+        run: |
+          docker build -t ghcr.io/kleveross/klever-web:$RELEASE_VERSION -f Dockerfile .
```

Comment:
Should set env RELEASE_VERSION like https://github.com/kleveross/klever-model-registry/blob/master/.github/workflows/publish.yml#L28